rkt/run.go

```diff
@@ -92,6 +94,11 @@ func runRun(args []string) (exit int) {
 		return 1
 	}
 
+	if len(flagPodManifest) > 0 && (len(flagVolumes) > 0 || len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || rktApps.Count() > 0 || flagLocal) {
```

Comment:
how did you decide on this set? (e.g. why flagLocal?)
I think we should probably split it into a func actually